casual/casual-api/src/main/java/se/laz/casual/api/util/PrettyPrinter.java

```diff
@@ -9,21 +9,35 @@
 import java.math.BigInteger;
 import java.util.UUID;
 
+/**
+ * Prints formatted in casual format
+ */
 public final class PrettyPrinter
 {
     private PrettyPrinter()
-    {}
+    {
+    }
 
     public static String casualStringify(Xid xid)
     {
-        return String.format("%s:%s:%s",toHex(xid.getGlobalTransactionId()), toHex(xid.getBranchQualifier()), xid.getFormatId());
+        return String.format("%s:%s:%s", toHex(xid.getGlobalTransactionId()), toHex(xid.getBranchQualifier()), xid.getFormatId());
     }
 
     public static String casualStringify(UUID uuid)
     {
         return String.format("%x%x", uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
     }
 
+    public static String print(UUID correlationId, UUID execution)
```

Comment:
I'm not sure about the name print. It doesn't print anything. If anything I'd rather call it format since it outputs a formatted string

Reply:
sounds good to me, format that is